presto-main/src/main/java/com/facebook/presto/execution/SqlTaskExecution.java

```diff
@@ -1313,12 +1318,25 @@ private void checkLifespanCompletion(Lifespan lifespan)
             if (lifespan.isTaskWide()) {
                 return; // not a driver group
             }
+
             if (!isNoMoreDriverRunners(lifespan)) {
                 return;
             }
+
+            // do we still have running tasks?
             if (getRemainingDriver(lifespan) != 0) {
                 return;
             }
+
+            // no more output will be created
+            outputBuffer.setNoMorePagesForLifespan(lifespan);
+
+            // are there still pages in the output buffer
```

Comment:
nit: `?` at the end

Reply:
@nezihyigitbasi : Fun fact, these comments are copied from 

https://github.com/prestodb/presto/blob/928eaa921ac3004fc5fa57a27823995487f18ae1/presto-main/src/main/java/com/facebook/presto/execution/SqlTaskExecution.java#L624-L651

So we should also add `?` there as well :P

Reply:
tough question :)